### PR TITLE
CIV-12435 Claimant settle confirmation page

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
**Before creating a pull request make sure that:**

[- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

](https://tools.hmcts.net/jira/browse/CIV-12435)
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by ElifDenizEsea — DISMISSED** [review]

File: src/main/common/models/claim.ts [hunk 1]
```diff
@@ -896,6 +896,10 @@ export class Claim {
     return this.claimantResponse?.courtDecision === RepaymentDecisionType.IN_FAVOUR_OF_CLAIMANT;
   }
 
+  hasClaimantAcceptedToSettleClaim(): boolean {
+    return this.isFullDefence() && this?.applicant1PartAdmitIntentionToSettleClaimSpec === YesNoUpperCamelCase.YES;
```
Comment (paul-pearson) [inline src/main/common/models/claim.ts:900]: Can `this` ever  be null?

File: src/main/common/models/claim.ts [hunk 2]
```diff
@@ -896,6 +896,10 @@ export class Claim {
     return this.claimantResponse?.courtDecision === RepaymentDecisionType.IN_FAVOUR_OF_CLAIMANT;
   }
 
+  hasClaimantAcceptedToSettleClaim(): boolean {
```
Comment (paul-pearson) [inline src/main/common/models/claim.ts:899]: Can move this method to claimant.ts

File: src/main/common/models/claim.ts (hunk 1 above)
Comment (kalachandrasekar1) [reply src/main/common/models/claim.ts:900]: fixed

File: src/main/common/models/claim.ts (hunk 2 above)
Comment (kalachandrasekar1) [reply src/main/common/models/claim.ts:899]: using same method in claim.ts also

**Review by paul-pearson — DISMISSED** [review]

**Review by neeta-hmcts — DISMISSED** [review]

**Pull request merged**